### PR TITLE
Project creation

### PR DESCRIPTION
**Pull request by pachecowolox**
## Summary
- Completed the project setup

## Trello Card
https://trello.com/c/b09WD9OW/42-pull-request

### CODE REVIEW

File: .idea/.gitignore [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Default ignored files
```
Comment (Gianni-Nicolas) [inline .idea/.gitignore]: Everything related to the ".idea" folder should not be displayed. To avoid it let's add this folder to git ignore:

* .idea/

**Review by Gianni-Nicolas — CHANGES_REQUESTED** [review]
Great job 😄 , I only commented on one detail

File: training/.gitignore [hunk 2]
```diff
@@ -0,0 +1,37 @@
+HELP.md
```
Comment (marcegl91) [inline training/.gitignore]: These files should be in the root folder, not inside a `training` folder. Here you have added a folder called `training` where inside of it is the code of the project, but everything should go in the root level of the repository (the folder that is immediately created when you clone the project).

File: .idea/.gitignore [hunk 3]
```diff
@@ -0,0 +1,3 @@
...
+/shelf/
+/workspace.xml
```
Comment (marcegl91) [inline .idea/.gitignore]: The `.idea` folder and all of their files should not be added to the repository.

**Review by marcegl91 — CHANGES_REQUESTED** [review]

**Review by marcegl91 — APPROVED** [review]
Great job! :tada:

**Review by juanwongramirez — APPROVED** [review]

**Review by Gianni-Nicolas — APPROVED** [review]

**Pull request merged**